frontend/ui/screensaver.lua

```diff
@@ -463,6 +467,7 @@ function Screensaver:setup(event, event_message)
             self.screensaver_type = "random_image"
         end
     end
+    -- NB Kept around to support old settings.
```

Comment:
Oh, ok, this is our way to not migrate it. Fine.

Reply:
I could do it the other way if you prefer.

Reply:
No real preference.
I guess that if an image is indeed a document, we can set that document as the sleep screen.
While having an EPUB or PDF set as "image_file" would feel more odd.

It's just that we'll mostly use it with images, and setting that image (a document)'s cover feels stranger mentionning a non-image document's cover.